core/src/test/java/tech/tablesaw/joining/DataFrameJoinerPerformanceTest.java

```diff
@@ -62,6 +62,26 @@ public void innerJoinCustomersFirst() {
     assertTimeout(Duration.ofSeconds(1), () -> customers.joinOn("customerId").inner(orders));
   }
 
+  @Test
+  public void innerJoinCustomersFirstTimeMeasure() {
```

Comment:
I'm not really sure what to do with this test, but I think you should disable it. We don't want to write (more) text strings to System.out. I spent a lot of time removing many that I put in myself. I would suggest using assertions rather than print statements, but I'm concerned that they will fail intermittently given the non-deterministic nature of performance tests. 

As long as you demonstrate correctness, and we "know" it improves performance, we don't need to run this every time. Also, any test that takes a long time to run is likely to break the CI runs.

Reply:
Yeah. I agree. I do not want to use assertion to do non-deterministic tests either. It is unstable.
I will delete this test in my PR. Maybe In the future we can find better ways to do performance improvement and test.
Thank you for your careful review : )